link/iter.go

```diff
@@ -78,6 +78,10 @@ func (it *Iter) Open() (io.ReadCloser, error) {
 	return fd.File("bpf_iter"), nil
 }
 
+func (it *Iter) Info() (*Info, error) {
```

Comment:
Delete this. Fall back to RawLink Info() method. Only metadata is not supported. Same for RawTracepointType and UprobeMultiType.